docs/docs/reference/crd-reference/taskdefinition.md

```diff
@@ -171,16 +167,31 @@ and Keptn sets up the container and runs the script as part of the task.
     for permissions and importing data
     so a script that works properly elsewhere
     may not function out of the box when run in the `deno-runtime` runner.
+    In this case you may want to use a custom container instead. 
 
     ```yaml
     {% include "../../assets/crd/examples/synopsis-for-deno-runtime-container.yaml" %}
```

Comment:
it seems like `synopsis-for-deno-runtime-container.yaml` does not exist - should we add the following here?

```
??? example 
         ```yaml
         {% include "../../assets/crd/deno-context.yaml" %}
         ```
```

Reply:
nevermind, just saw that this file in fact does exist, sorry for the confusion